css/styles.css

```diff
@@ -3,57 +3,144 @@
 }
 
 .container {
-  width: 1080px;
+  width: 960px;
   margin: 0 auto;
   height: 100vh;
 }
 
-.tagline {
-  border: 1px solid black;
-  height: 100px;  
+/* Header */
+header {
+  border-bottom: 3px solid yellow;
+  background-color: black;
+  color: white;
+  padding: 1px 30px;
 }
 
-header {
-  border: 1px solid black;
-  background-color: green;
+header h1 {
+  color: yellow;
+  font-size: 35px;
+  font-family: 'Merriweather';
```

Comment:
@dylantg you should always apply font fallback incase the fonts that you are requesting doesn't load in a timely manner or fails.